explorer/app/apis/azul/hca-dcp/common/aggregatedEntities.ts

```diff
@@ -0,0 +1,8 @@
+export interface AggregateProject {
```

Comment:
Would you be able to update the naming here to `AggregatedProject` and `AggregatedProjectResponse`?